packages/@react-aria/utils/src/getScrollParent.ts

```diff
@@ -12,7 +12,10 @@
 
 import {isScrollable} from './isScrollable';
 
-export function getScrollParent(node: Element, checkForOverflow?: boolean): Element {
+export function getScrollParent(node: Element | null, checkForOverflow?: boolean): Element {
```

Comment:
I'd say usually we should not accept null as input, we should check it at the usage site when needed.